doc/modules/covariance.rst

```diff
@@ -24,22 +23,22 @@ Empirical covariance
 ====================
 
 The covariance matrix of a data set is known to be well approximated
-with the classical *maximum likelihood estimator* (or "empirical
+by the classical *maximum likelihood estimator* (or "empirical
 covariance"), provided the number of observations is large enough
 compared to the number of features (the variables describing the
 observations). More precisely, the Maximum Likelihood Estimator of a
-sample is an unbiased estimator of the corresponding population
+sample is an unbiased estimator of the corresponding population's
 covariance matrix.
 
 The empirical covariance matrix of a sample can be computed using the
 :func:`empirical_covariance` function of the package, or by fitting an
 :class:`EmpiricalCovariance` object to the data sample with the
-:meth:`EmpiricalCovariance.fit` method.  Be careful that depending
-whether the data are centered or not, the result will be different, so
-one may want to use the ``assume_centered`` parameter accurately. More precisely
-if one uses ``assume_centered=False``, then the test set is supposed to have the
-same mean vector as the training set. If not so, both should be centered by the
-user, and ``assume_centered=True`` should be used.
+:meth:`EmpiricalCovariance.fit` method. Be careful that results depend
+on whether the data are centered, so one may want to use the
```

Comment:
"Data" is the plural of datum. According to [merriam-webster](https://www.merriam-webster.com/dictionary/data), though:

> "Both constructions are standard. The plural construction is more common in print, evidently because the house style of several publishers mandates it. "

Reply:
I know, it was just a side comment, it's fine the way it is.